test/jasmine/assets/delay.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/**
+ * This is a very quick and simple promise delayer. It's not full-featured
+ * like the `delay` module.
+ *
+ * Promise.resolve().then(delay(50)).then(...);
+ */
+module.exports = function delay(duration) {
```

Comment:
I debated just adding [`delay`](https://www.npmjs.com/package/delay) as a proper dev dependency. It handles arguments etc. more carefully than this quick hack, but this was all I needed.